openra/classes/file_location.py

```diff
@@ -18,6 +20,16 @@ def __init__(self, fs: FS, path: str, file: str):
         self.path = path
         self.file = file
 
+    def get_filename_without_extension(self):
+        return os.path.splitext(self.file)[0]
+
+    def get_file_extension(self):
+        extension = os.path.splitext(self.file)[1]
+        if (len(extension) == 0):
```

Comment:
unnecessary parenthesis around the if statement.

Reply:
Ok I've taken out the extra brackets